### ISSUE TITLE
A problem occurred evaluating script. > For input string: "APP_VERSION_CODE"

### ISSUE THREAD

**Issue by DilnaMohandas** [Sunbird-Ed/SunbirdEd-mobile#254]
What went wrong:
A problem occurred evaluating the script.
> For input string: "APP_VERSION_CODE"

This error is thrown while trying to run the app in android using the command ionic cordova run android.

**Comment by swayangjit** [comment]
Hi Dilna ,
In subird.properties file app_version_code should  be like following
`app_version_code = 1`
Let us confirm if that's working for you.

**Comment by DilnaMohandas** [comment]
Yes, this worked, but now the same issue with dev_app_id, what should be the value for dev_app_id?
I tried giving org.sunbird.app but it is throwing : 
sunbird-mobile/platforms/android/app/build/intermediates/manifests/full/dev/debug/AndroidManifest.xml:2: error: attribute 'package' in <manifest> tag is not a valid Android package name:"org.sunbird.app"

**Comment by DilnaMohandas** [comment]
Yes, this worked. Thanks
